### ISSUE TITLE
A VF "terminology directory"

### ISSUE THREAD

**Issue by gcassel** [valueflows/valueflows#119]
I think this idea has come up, but I haven't seen an existing Issue for the subject of a VF dictionary/glossary/encyclopedia/whatever.

I'm picturing the potential value of a central "terminology directory".  The directory could include direct descriptions of some terms, but link to official documents for other terms.  (Some terms seem to already have literally *definitive* md documents, distributed across several repositories.)

The directory itself would be a new doc, and could be linked in VF's main [README](https://github.com/valueflows/valueflows/blob/master/README.md)  and/or the (maybe irrelevant?) wiki.  

Any thoughts?  

I haven't authored a Github doc before, but if people like the idea of a terminology directory, this could be a good place for me to start!

**Comment by fosterlynn** [comment]
A scope question: Would this cover the vocabulary itself or more or less? 

I like the idea of a central repository for the vocabulary definitions, and maybe more.  And I like the idea of you starting one @gcassel .  :)

**Comment by gcassel** [comment]
> A scope question: Would this cover the vocabulary itself or more or less? 

This would not cover the development of the vocabulary.  I don't think that would be good modular design.   

If the "terminology directory" does grow to include some direct definitions for minor terms, I think that those should be added by pull requests.   

Does that answer your question @fosterlynn ?  :)

**Comment by elf-pavlik** [comment]
In many cases I like using wiki approach to .md pages in repositories "just create new page, later others can edit it or you can delete it".

Some inspiration: https://www.w3.org/TR/ld-glossary/

**Comment by gcassel** [comment]
> In many cases I like using wiki approach to .md pages in repositories "just create new page, later others can edit it or you can delete it".

Yeah I think that's great.  I believe the main purpose of the glossary/ terminology directory will be simply to point people at each relevant .md page.   

It will probably be best to just stick with links to .md files at first, and see how that goes.   Not sure yet how much terminology any of us will want to define.

Thanks for the good link!

**Comment by fosterlynn** [comment]
@gcassel all cool.  I think you should go for it!  It can develop organically, as people like to say these days, put something out there and see what people think!

> Not sure yet how much terminology any of us will want to define.

We talk about definitions all the time, so I'd say we do want to define it, and it would be great to have someone capturing and curating that!!

**Comment by elf-pavlik** [comment]
another example of terminology page, this one focused on food networks https://github.com/ouisharelabs/food-dashboard/blob/master/TERMINOLOGY.md
